content/tutorials/single-page-tutorials/proof-inclusion-of-a-block.md

```diff
@@ -0,0 +1,372 @@
+# Tokenless Data Notarization
```

Comment:
Btw, is metadata left out intentionally or did we just forgot to add it?

@Dr-Electron

Reply:
I guess we just forgot it. Not sure if @adrian-grassl even knows metadata 🤔

Reply:
I think we can update this in a new PR if needed